### PR TITLE
WINC-551 New auto-scaling feature for historical tracking

### PR DESCRIPTION
**Pull request by codyhoag**
Release note for auto-scaling Windows instances for previous release.

Preview: https://deploy-preview-37928--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-2-x.html#wmco-2-0-1-autoscaling-windows-instances

### CODE REVIEW

**Comment by codyhoag** [comment]
@sgaoshang can you verify this new feature that was missed in the WMCO 2.x release notes? Thanks!

File: windows_containers/windows-containers-release-notes-2-x.adoc [hunk 1]
```diff
@@ -64,6 +64,15 @@ This release adds the following new features and improvements.
...
 Image pull time-out has been increased to 30 minutes.
 
+[id="wmco-2-0-1-autoscaling-windows-instances"]
+==== Autoscaling for Windows instances
+
+Cluster autoscaling is now supported for Windows instances. You can complete the following actions for Windows nodes:
+
+* Define and deploy a xref:../machine_management/applying-autoscaling.adoc#configuring-clusterautoscaler[cluster autoscaler].
+* Create a Windows node using a machine set.
+* Define and deploy a xref:../machine_management/applying-autoscaling.adoc#configuring-machineautoscaler[machine autoscaler], referencing a Windows machine set.
```
Comment (mburke5678) [inline windows_containers/windows-containers-release-notes-2-x.adoc:74]: My ignorance. Is there a difference between a _machine set_ and a _Windows machine set_?

Should we link to machine set in the second bullet for consistency, assuming there is a proper location to link to?

**Comment by mburke5678** [comment]
@codyhoag A couple of pesky questions. Otherwise LGTM.

File: windows_containers/windows-containers-release-notes-2-x.adoc (hunk 1 above)
Comment (codyhoag) [reply windows_containers/windows-containers-release-notes-2-x.adoc:74]: There is a difference between a machine set and a Windows machine set. I will add the link there for consistency :+1: Thanks!

**Pull request merged**